src/interfaces/breadcrumb-interface.ts

```diff
@@ -1,7 +1,5 @@
 export interface BreadcrumbIF {
```

Comment:
interfaces/index.ts should import the shared IF instead of using this file, right?

Reply:
mmm yea! You are right, we have access to that now if i update the imported IF version.

Reply:
oof, when i update the IF's version something goes gravely wrong with todolist and todolist.spec interfaces. Looking into it 👀

Reply:
I have no idea the relation to the shared components Interfaces and the TODO Component.

Reply:
These?
https://github.com/bcgov/business-filings-ui/blob/fdf168ad3036f7459bb601c398d5532724081803/src/components/Dashboard/TodoList.vue#L541

Reply:
It's specifically the ConfirmDialogType, it is somehow breaking that component. 
When the shared Interfaces are used in a ts file, no issues, but when used directly on a Vue Component causing issues.

Reply:
This is tech debt associated with the updated shared interfaces ... I sympathize ... but it would be better to not have multiple copies of this IF and to not carry forward more tech debt.

Reply:
Weird, the ConfirmDialogType is the same in Filings UI as in the shared library.

Reply:
I'm wondering if it's related to the Vue version used by the shared interface vs our vue version locally

Reply:
I'd be surprised. Basically Filings UI imports the code for the shared component and compiles it using Filing UI's Vue libraries.

Reply:
Updated Enum version as well 👍